gen/teams_provisioning

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/perl
+use strict;
+use warnings;
+use perunServicesInit;
+use perunServicesUtils;
+use Perun::Agent;
+use JSON::XS;
+use Data::Dumper;
+
+our $SERVICE_NAME = "teams_provisioning";
+our $PROTOCOL_VERSION = "3.0.2";
+my $SCRIPT_VERSION = "3.0.2";
```

Comment:
Original version is `3.0.0` for this service so you should use `3.0.1` now. Also protocol version should stay at `3.0.0` since content of the generated files has not changed.